packages/notification-service/src/blockscout/transfers.ts

```diff
@@ -147,12 +157,20 @@ export async function handleTransferNotifications(): Promise<void> {
     latestBlock: goldTransfersLatestBlock,
   } = await getLatestTokenTransfers(goldTokenAddress, blockToQuery, Currencies.GOLD)
 
+  const {
+    transfers: nativeGoldTransfers,
+    latestBlock: nativeGoldTransfersLatestBlock,
+  } = await getLatestTokenTransfers(goldTokenAddress, blockToQuery, Currencies.GOLD, false)
```

Comment:
Could you leave a comment about why we using this additional fetch which doesn't use the logs query?